src/studio/kdb/Config.java

```diff
@@ -94,6 +94,32 @@ private void init() {
             return;
         }
 
+        if (!Files.exists(file) && System.getProperty("os.name").startsWith("Windows")) {
+            System.out.println("Config not found in userprofile. Trying legacy path.");
+            //Old Java versions returned a different place for user.home on Windows.
+            //A user upgrading from such old directory would suddenly "lose" their config.
+            String oldpath = null;
+            try {
+                Process process = Runtime.getRuntime().exec("reg query \"HKCU\\Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Shell Folders\" /v Desktop");
+                BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+                String line = null;
+                while ((line = reader.readLine()) != null) {
+                    if (line.contains("Desktop") && line.contains("REG_SZ")) {
```

Comment:
I think the intent might be clearer using regular expressions, but looks like it should work.